inc/osvr/Util/Logger.h

```diff
@@ -203,7 +203,7 @@ namespace util {
             OSVR_UTIL_EXPORT void flush();
 
             /// Get the logger name
-            OSVR_UTIL_EXPORT std::string const &getName() const {
```

Comment:
Why should we not export `Logger::getName()`?

Reply:
it's inline

Reply:
Right. I think we addressed all the Logger export stuff in a separate PR already.